README.md

```diff
@@ -46,11 +46,12 @@ Since version 1.7.0 we publish the [`puppeteer-core`](https://www.npmjs.com/pack
 a version of Puppeteer that doesn't download Chromium by default.
 
 ```bash
-npm i puppeteer-core
-# or "yarn add puppeteer-core"
+npm i puppeteer-core@chrome-71
+# or "yarn add puppeteer-core@chrome-71"
```

Comment:
I'd keep this as, otherwise it is confusing: reader at this point has no idea what's `chrome-71` and why they should install this version of pptr-core.